Source/WebCore/platform/graphics/texmap/FloatPolygon3D.cpp

```diff
@@ -49,14 +51,38 @@ FloatPolygon3D::FloatPolygon3D(const FloatRect& rect, const TransformationMatrix
         m_normal = edge1.cross(edge2);
     }
     m_normal.normalize();
+
+    computeBounds();
 }
 
 FloatPolygon3D::FloatPolygon3D(Vector<FloatPoint3D>&& vertices, const FloatPoint3D& normal)
     : m_vertices(WTFMove(vertices))
     , m_normal(normal)
 {
+    computeBounds();
 }
 
+void FloatPolygon3D::computeBounds()
```

Comment:
Why don't we compute bounds on-demand? Any reason to store it?

Reply:
FloatPolygon3D::computeBounds() removed